### ISSUE TITLE
Optional contrails on weapons and aircraft

### ISSUE THREAD

**Issue by seaben** [OpenRA/OpenRA#18242]
## Motivation
I like Tiberian Dawn and Red Alert mainly because of the art style and the atmosphere it creates.

These "contrails" added by the mods in OpenRA are hurting my experience. This goes so far, that I am discouraged to play the game at all, when I cannot deactivate the contrails by changing the trail length to zero in the aircraft and weapons YAML files. For example, this is the case when I install the game via Snap.

## Proposed solution
I would much appreciate you considering to add an option in the game to deactivate all contrails on weapon and aircraft animations.

## Side effects
Some might see it as a disadvantage for a player that does not get the clear information about traveling shots in a game.

So you might add a general option that a player can use to deactivate contrails and may be at a disadvantage in a match. Or you might provide an option in the game lobby, so all players have a consistent setting. I would prefer the former, since the preference is based on subjective perception of the art and the mentioned disadvantage is also subjective. But I would be glad in any case that I can enjoy the game with less effort and limitation.

Thank you for reading.

**Comment by MlemandPurrs** [comment]
Those on the Aircraft can be made to stick out less via
```
		TrailLength: 8
		TrailWidth: 48
		UsePlayerColor: false
```
Yak's needs to remain at TrailLength: 10 to keep connect properly to body.
As for Missiles can give ContrailLength: 8

**Comment by pchote** [comment]
I wouldn't object in principle to having an option in the settings to disable these.

**Comment by seaben** [comment]
Excuse me for commenting again, but I want to clarify: I am mainly interested in having this feature in Tiberian Dawn, as I prefer it because of the concurrent production. So if you decide in favor of the feature, please make it available in TD.

**Comment by penev92** [comment]
I believe I have seen previous suggestions to implement a "reduced effects" option or a set of graphics options to enable/disable different things like that.
If we could get a few other things like contrails, that would make this more worthwhile.

**Comment by matjaeck** [comment]
#19872

**Comment by matjaeck** [comment]
@Mailaender Please elaborate why you keep hiding my comments. This is not the first occasion. If my comments or presence are deemed a violation of the CoC (which I think was invalidated by an unreviewed commit of you some months ago but let's assume it still stands) I demand to be banned from this repository.

This abusement of write access, be it unreviewd changes to the CoC or hiding comments without reason is just pathetic and a disgrace for this project and what it claims to be.

**Comment by abcdefg30** [comment]
What is the relation between this ticket and #19872?

**Comment by matjaeck** [comment]
> What is the relation between this ticket and #19872?

I disagree with the implication that a missing relation is justification for this repeated abuse of mod tools. It is not common in this repository to hide unrelated comments and  these actions are specifically aimed at me from the person who keeps abusing their write access (another example is the sposors-branch disgrace.

To answer your question, I understand this issue as a location to collect items that may interfere with art style and the atmosphere of original games. The added on-screen text notifications do affect the look and feel of the game. The feature itself is great.

**Comment by matjaeck** [comment]
Since I do find this ticket important and dont want to derail it, im now going to delete these offtopic comments, knowing they are logged and read in other places (and mostly a waste of time).
